batch/test/test_scale.py

```diff
@@ -16,24 +14,15 @@ def client():
 
 
 def test_scale(client):
-    now = time.time()
-    n_batches = 100
-    batches = []
-    for _ in range(n_batches):
-        n_jobs = 100
-        batch = client.create_batch()
-        for idx in range(n_jobs):
-            sleep_time = random.uniform(0, 30)
```

Comment:
Let's just delete this file. It's not used and I can write my own script if needed.

Reply:
If you do that, then you can delete the exclusions from all of `test_batch_*` in build.yaml

Reply:
I reverted this, let's leave this change for a different PR.